tailscale.sh

```diff
@@ -38,53 +38,36 @@ tar xzf tailscale.tgz
 tar_dir="$(echo ${tarball} | cut -d. -f1-3)"
 test -d $tar_dir
 
-# create our target directory structure
-mkdir -p tailscale/usr/{bin,sbin,lib/{systemd/system,extension-release.d}}
+# Create binaries directory in home
+mkdir -p /home/deck/.bin
 
-# pull things into the right place in the target dir structure
-cp -rf $tar_dir/tailscale tailscale/usr/bin/tailscale
-cp -rf $tar_dir/tailscaled tailscale/usr/sbin/tailscaled
+# pull binaries
+cp -rf $tar_dir/tailscale /home/deck/.bin/tailscale
+cp -rf $tar_dir/tailscaled /home/deck/.bin/tailscaled
```

Comment:
I wonder if we should put these in a subfolder, to keep things tidy?

Reply:
Also, one thing I just realised: https://github.com/saumya-banthia/tailscale-control/blob/main/main.py uses Python's `subprocess.run`, which uses sh instead of bash, which means it won't be able to find the `tailscale` binary

It's an easy fix though, we'll just need to notify the maintainer of that repo if we decide to go ahead with this solution

Reply:
Do you know why the following works, yet the plugin is still not functional with the current changes? :
```bash
$ /bin/sh
sh-5.1$ python
>>> import os
>>> import subprocess
>>> subprocess.run(["tailscale"])
USAGE
  tailscale [flags] <subcommand> [command flags]
...
>>>
```

Reply:
I'm not sure. If I had to guess, I'd say running sh via an interactive shell works differently to how Decky runs it, and causes the $PATH variable to not be defined in the same way

Reply:
In any case I've left a potential solution in https://github.com/tailscale-dev/deck-tailscale/issues/13#issuecomment-1858974758, I'll try it out